packages/front-end/components/Features/SDKConnections/SDKConnectionsList.tsx

```diff
@@ -40,17 +43,79 @@ export default function SDKConnectionsList() {
     project
   );
 
+  const useNewEmptyStateLayout = useFeatureIsOn(
+    "sdk-connections-new-empty-state"
+  );
+
+  const popularLanguages = getLanguagesByFilter("popular");
```

Comment:
@jdorn I am using `popular` here to keep it in sync with the options shown when the modal is first opened. But we can also hardcode the list for this layout if that's preferred.

Reply:
I'm worried someone using Golang for example, would just abandon if they saw this because there's no indication that we support more than what's listed.

A few options:
1. Show all languages, but group by category so it's not one huge list
2. Add tabs for the different categories and let the user switch between them (like we do in the modal)
3. Add a "See more..." link that reveals the rest of the languages when clicked

Reply:
@jdorn Updated the layout to use option 3 & updated the gif in the description with the new experience.